qiita_pet/support_files/doc/source/tutorials/prepare-information-files.rst

```diff
@@ -5,10 +5,30 @@
 Prepare information files
 =========================
 
-The set of required fields for the *sample information file* and *prep
-information file* files varies based on the functionality that you want to
+The set of required fields for the *sample information* and *preparation
+information files* varies based on the functionality that you want to
 use from the system.
 
+As described in :ref:`../qiita-philosophy/`, a Qiita study can have
```

Comment:
@antgonza I'm having trouble connecting these files using the Sphinx references, instead of html links. I'm not familear with Sphinx to try some of these [proposed solutions](https://stackoverflow.com/questions/10199233/can-sphinx-link-to-documents-that-are-not-located-in-directories-below-the-root).